tests/common_tests/helpers/generate_test_tpc.py

```diff
@@ -205,7 +205,9 @@ def generate_test_attr_configs(default_cfg_nbits: int = 8,
                                kernel_cfg_nbits: int = 8,
                                kernel_cfg_quantizatiom_method: QuantizationMethod = QuantizationMethod.POWER_OF_TWO,
                                enable_kernel_weights_quantization: bool = True,
-                               kernel_lut_values_bitwidth: int = None):
+                               kernel_lut_values_bitwidth: int = None,
+                               enable_bias_weights_quantization: bool = False,
```

Comment:
this shouldn't be added to a utility test tpc function, but setup in the specific test that uses it